news/h5py.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* Documentation-level fix.
```

Comment:
Should this be a fixed rather than an add?

Reply:
Yes